src/main/java/org/jenkinsci/plugins/awsdevicefarm/AWSDeviceFarmRecorder.java

```diff
@@ -550,11 +565,17 @@ public void perform(@Nonnull hudson.model.Run<?, ?> build, @Nonnull FilePath wor
                     }
                 }
             }
-
             // Get AWS Device Farm project from user provided name.
             writeToLog(log, String.format("Using Project '%s'", projectName));
             Project project = adf.getProject(projectName);
 
+            VpcConfig vpcSettings = project.getVpcConfig();
```

Comment:
Can we test this using an AWS account with VPC enabled and not?

Reply:
I am not sure about account level VPC, but I did test this a projects that both had vpc and did not have vpc